src/analyses/call_graph.h

```diff
@@ -20,23 +20,33 @@ Author: Daniel Kroening, kroening@kroening.com
 class call_grapht
 {
 public:
-  call_grapht();
-  explicit call_grapht(const goto_modelt &);
-  explicit call_grapht(const goto_functionst &);
+  explicit call_grapht(bool collect_callsites=false);
+  explicit call_grapht(const goto_modelt &, bool collect_callsites=false);
```

Comment:
Safe to drop `explicit`

Reply:
Not so -- with a default argument there are one-arg versions of both constructors.